### DISCUSSION TITLE
M1 installs gone awry

### DISCUSSION THREAD

**Discussion by capwalker** [Homebrew/discussions#1324] [Getting started]
I've been lucky to add an M1 Mac Mini to my network. I'd gotten enough spurious errors that I wanted to install for the new processor. Needless to say a newbie and a complex error signals "call on the experts". But I wanted to remove it all, then do one last install.  Now it works and "BREW DOCTOR" says good to go.

**Comment by SMillerDev** [comment]
I'm not sure what you want help with

**Comment by capwalker** [comment]
In that post I could have said "If you. have any issues with an M1 chip and
Homebrew , try completely deleting it then reinstall as normal. I just did
that and my issues went away.
